scripts/package-binary

```diff
@@ -8,3 +8,4 @@ source ./scripts/version.sh
 mkdir -p dist/artifacts
 
 install -s bin/${PROG} dist/artifacts/${RELEASE}
+install -s bin/${PROG}.exe dist/artifacts/${RELEASE}
```

Comment:
Should this have some guard to only run on Windows?

Reply:
This one is weird to me. The reason for my original use of `install` is that I wanted correct file mode/perms and symbol stripping in a single operation. Does this even work on Windows?